### ISSUE TITLE
Accessibility doesn't work with the Drawer

### ISSUE THREAD

**Issue by MaxToyberman** [root-two/react-native-drawer#332]
My app supports Accessibility, but when the Drawer is opened everything behind the drawer is being focused (Drawer behavior should be like a Modal behavior only the Drawer should be focused) 
![drawer](https://user-images.githubusercontent.com/11058371/31895508-042ce22c-b81a-11e7-9302-e5e4520df534.jpeg)

**Comment by SuhairZain** [comment]
@MaxToyberman Have you found a solution to this issue?

**Comment by MaxToyberman** [comment]
@SuhairZain  Actually I found a solution. 
**_on Android you have 2 possible solutions:_**
1) you can use `DrawerLayoutAndroid` from react-native ,which is the native android component,(you don't need to anything there) it just works.
[](https://facebook.github.io/react-native/docs/drawerlayoutandroid.html)

2)You can give the View behind the drawer the `importantForAccessibility` property
like this : 
```
class HomePage extends Component {
    constructor(props) {
        super(props);
        this.state = { 
            importantForAccessibility: "auto"
         }
     }
<View importantForAccessibility={this.state.importantForAccessibility}>

</View>
}
```
then, when you open your drawer you do `this.setState({importantForAccessibility:"no-hide-descendants"})`

when you close the drawer you do `this.setState({importantForAccessibility:"auto"})` again.

according to the docs this will force accessibility services to ignore the component and all of its children.

**_on Ios the solution is very simple:_**
Just give your Drawer accessibilityViewIsModal

I have issued a PR for ios you can see it here 
`https://github.com/root-two/react-native-drawer/pull/351`

I hope it helps
